docs/UserGuide.md

```diff
@@ -158,147 +128,241 @@ Examples:
 
 Success Output:
 * For the command `list` or `list schedule`:
-`Showing list SCHEDULE`
-![Success for list SCHEDULE](images/list/list_schedule_positive.png)
+  `Showing list SCHEDULE`
+  ![Success for list SCHEDULE](images/list/list_schedule_positive.png)
 * For the command `list students` (including extra keywords):
-`Showing list STUDENT`
-![Success for list STUDENTS](images/list/list_student_positive.png)
+  `Showing list STUDENT`
+  ![Success for list STUDENTS](images/list/list_student_positive.png)
 * For the command `list tasks`
   `Showing list TASK`
-![Success for list STUDENTS](images/list/list_tasks_positive.png)
+  ![Success for list STUDENTS](images/list/list_tasks_positive.png)
 
 Notes
 * If there are no entries, e.g. there are no students added yet or there are no tasks added yet, an empty list is displayed.
-![Empty list](images/list/list_tasks_empty.png)
+  ![Empty list](images/list/list_tasks_empty.png)
 
 
 Failure Output:
-* When there are invalid keywords specified as a parameter: 
+* When there are invalid keywords specified as a parameter:
 ```Invalid command format!
-  list: Displays the specified list, which can be a STUDENTS list, SCHEDULE list or TASKS list. Default command without specified list displays the schedule list. When specifying STUDENTS list,optional parameters can be used to specify what student details to display
+  list: Displays the specified list, which can be a STUDENTS list, SCHEDULE list or TASKS list. Default command without specified list displays the schedule list. When specifying STUDENTS list, optional parameters can be used to specify what student details to display.
   Parameters: [LIST] [KEYWORDS]...
   Example: list SCHEDULE
   Example: list STUDENTS phone email
-  Example: list TASKS`
+  Example: list TASKS
+```
+
+### Showing a lesson's/task's/student's details : `show`
+
+The show command has different behaviours depending on the current list:
+
+1. In `SCHEDULE` list:
+    - Shows the details of the specified lesson from the schedule list in the application. 
+2. In `TASKS` list:
+    - Shows the details of the specified task from the full task list in the application.
+3. In `STUDENTS` list:
+    - Shows the details of the specified student from the contact list in the application.
+
+Format: `show INDEX`
+
+* Shows the details of the lesson/task/student at the specified `INDEX`.
+* The index refers to the index number shown in the displayed schedule/task/student list.
+* The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
Since some nerds test with max int ( and probably in week 13, some will test with number that is larger than max int, 1000000000000000 for example), I will modify the behavior of the parser such that can take at most 5 consecutive integer (00000 - 99999).

Reply:
LOL okies will update this accordingly 😃

Reply:
Made the necessary changes in recent commit. Thank you!